pymfe/clustering.py

```diff
@@ -300,14 +330,27 @@ class (effectively holding the same result as if the argument
         precomp_vals = {}
 
         if not {"representative"}.issubset(kwargs):
-            precomp_vals["representative"] = (
-                MFEClustering._get_class_representatives(
-                    N=N, y=y, representative=representative, classes=classes))
+            precomp_vals["representative"] = cls._get_class_representatives(
+                N=N, y=y, representative=representative, classes=classes)
 
         return precomp_vals
 
+    @staticmethod
+    def _calc_cls_inds(y: np.ndarray, classes: np.ndarray) -> np.ndarray:
```

Comment:
It is true. However in this case there is a problem of cyclic dependence between the two modules (the clustering module is already imported inside the complexity module.) In Python, it is possible to verify if a module was previously imported to avoid errors related to this issue, but still it is not ideal in my opinion. To fix this problem, and avoid future problems similar to that one, maybe we could create a _utils.py module to keep "sufficiently generic auxiliary" shared functions, like this one?

Reply:
I agree with you. So, I think option 1) is better. What do you think?

Reply:
The first option is good for me too. Also, I think it may scales better.